app/nlg/country.py

```diff
@@ -84,28 +86,23 @@ def render(cnc, lexeme, entity):
 
 	region_nps  = []
 	region_advs = []
-	preposition = w.of_1_Prep if cnc.name in ["ParseSpa", "ParseFre"] else w.in_1_Prep
```

Comment:
This line should stay otherwise French/Spanish will break. For Russian the if statement will give you in_1_Prep which is what you want